host/src/containers/GameSessionContainer.tsx

```diff
@@ -157,23 +157,23 @@ const GameSessionContainer = () => {
         setGameSession(response);
         checkGameTimer(response);
 
-  
+
       });
   };
 
   const checkGameTimer = (gameSession) => {
-      if (gameSession.currentState !== GameSessionState.CHOOSE_CORRECT_ANSWER && gameSession.currentState !== GameSessionState.CHOOSE_TRICKIEST_ANSWER){
-          setGameTimer(false);
-          setGameTimerZero(false);
-      }
-      else {
-          setGameTimer(true);
-          setGameTimerZero(false);
-      }
+    if (gameSession.currentState !== GameSessionState.CHOOSE_CORRECT_ANSWER && gameSession.currentState !== GameSessionState.CHOOSE_TRICKIEST_ANSWER) {
+      setGameTimer(false);
+      setGameTimerZero(false);
+    }
+    else {
+      setGameTimer(true);
+      setGameTimerZero(false);
+    }
   };
 
   const handleStartGameModalTimerFinished = () => {
```

Comment:
I think we can clean this function up a bit based on your changes to `handleStartGame.` 

Currently, we go `handleStartGame` -> Timer -> `handleStartGameModaltimerFinished` and in that last function that's where we update the `gameSession.` With your changes, we are making two calls to `apiClient.updateGameSession` with similar information and it'd be nice to trim that down to one. It might make sense to move all this logic (including the `teamDataRequests` to the `handleStartGame` function and then just close the modal here. I'm guessing you might be able to remove this function altogether and just close the modal directly with setIsModelOpen(false) from the modal. 

I think this is something that will have to be verified. But the original code prior to your PR is a bit messy and it'd be great to get it simplified here. 

Timer function looks good now though. I don't think there's an issue with the slight difference in values; that's just a function of one app driving the other imo